src/darwin/frida-helper-backend-glue.m

```diff
@@ -1843,6 +1845,16 @@ static void frida_darwin_helper_backend_launch_using_sbs (NSString * identifier,
 
   dyld = gum_darwin_module_new_from_memory ("/usr/lib/dyld", task, dyld_header, GUM_DARWIN_MODULE_FLAGS_NONE, NULL);
   instance->dyld = dyld;
+  instance->dyld_size = 0;
+  i = 0;
+  while ((segment = gum_darwin_module_get_nth_segment (dyld, i++)) != NULL)
+  {
+    if (strncmp (segment->name, "__TEXT", sizeof (segment->name)))
```

Comment:
```suggestion
    if (strcmp (segment->name, "__TEXT") == 0)
```

This would work too right? And I guess this logic was inverted since it was checking for non-zero?

Reply:
wow yeah 🤔